### PR TITLE
Code coverage of react code with Istanbul and jest

### PR DESCRIPTION
**Pull request by oskarv**

### CODE REVIEW

**Comment by oskarv** [comment]
Working on the wiki page

**Comment by oskarv** [comment]
Wiki page is done: https://github.com/oskarv/spring-react/wiki/Code-coverage-reports-and-custom-configuration-with-Istanbul,-Jest-and-React

File: reactjs/.eslintrc.json [hunk 1]
```diff
@@ -6,7 +6,6 @@
     },
     "extends": [
         "standard",
-        "eslint:recommended",
```
Comment (illusi0n) [inline reactjs/.eslintrc.json]: why is this removed?

Comment (oskarv) [reply reactjs/.eslintrc.json]: it is obsolete, rules are completely covered with "plugin:react/recommended" extension

**Comment by oskarv** [comment]
Rodger that. I'll update the wiki
